examples/webpack.config.local.js

```diff
@@ -15,7 +15,7 @@ const ALIASES = require('ocular-dev-tools/config/ocular.config')({
 }).aliases;
 
 const ROOT_DIR = resolve(__dirname, '..');
-const PACKAGE_INFO = require(resolve(ROOT_DIR, 'package.json'));
+const LERNA_INFO = require(resolve(ROOT_DIR, 'lerna.json'));
```

Comment:
Oh, this was just an issue when running locally?

Reply:
`__version__` ends up always empty, so default to `latest` which is major/minor version.